<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Все категории</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>В мире</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Общество</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Происшествия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Армия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Экономика</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Культура</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Религия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Туризм</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Спорт</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Наука</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Образование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Политика</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ключевое слово:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Не сортировать</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>В алфавитном порядке (А-Я)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>В алфавитном порядке (Я-А)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По категориям</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По времени (сначала новые)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>По времени (сначала старые)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Категория:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>290</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>880</width>
      <height>171</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
